### ISSUE TITLE
[Getting Started] More prominent walkthroughs on Welcome

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode#122702]
**tl;dr:** Feature the core Getting Started guides with more prominent placement and design, to raise the likelihood that new and intermediate users discover them.

**Problems**

1. **Muted design causes the walkthroughs to go unnoticed.** While the first Getting Started page (_1.54_), featuring large cards for walkthroughs, had the highest engagement, adding back the well-known Welcome page to bring back file/folder/recent tasks lowered engagement significantly.
2. **Walkthroughs are interesting for more users than just first-time installs.** The _v1.54_ did open setup to all users until items got done, while the _1.55_ heuristic reduced it to users in their first day with VS Code. While the core audience for getting started is first-time users, we found that returning users used the walkthroughs to tweak their setup and deepen their skills.

**Options**

The assumption is that that we keep the Welcome page actions while elevating the walkthroughs. These placement ideas are just meant as conversation starters and to spawn more UX ideas:

1. Larger cards design for the top-most walkthrough
2. Expand top 3 walkthrough cards into larger cards
3. Dismissible gallery for 3 walkthroughs cards, above the welcome page content

![image](https://user-images.githubusercontent.com/8599/116634518-20841f80-a911-11eb-857e-ffeba589ee81.png)

_3. is my recommendation as it gives us the biggest impact to experiment with, applies the lesson that top-most welcome content gets most engagement and puts the trifecta of walkthroughs into one journey. As a bonus we can measure opt-out rates of users hiding the element, if we want decide to implement that toggle._

**Risks**

- How could we know about the negative impact of being distracting for existing users? Maybe track satisfaction, instrument a clear opt-out or combine existing telemetry.
- Depending on completion might make the placement stale. How can we make sure mostly partially done walkthroughs that users don't intent to finish make room for the next walkthrough.

cc @misolori @JacksonKearl

**Comment by JacksonKearl** [comment]
Didn't also change the focus-first-page behaviour from "when nothing is checked off" to "when nothing is checked off and it's the first session"? How do the new numbers compare to when focus-first was off entirely?

**Comment by digitarald** [comment]
Yes, I have that one the experiment analysis. Focus-setup-first did help with increasing engagement, but not having it wasn't a massive dip compared to 1.55 vs 1.54.

With #122570 more items should be checked off for returning users and we don't need the first-day check. I'll file another issue to remove the first-session heuristic afterwards.

**Comment by digitarald** [comment]
@misolori what are your UX thoughts on nudging up the visibility of walkthroughs to give more users a moment of awareness before they habitually moving onto recent projects or closing the welcome page?

**Comment by misolori** [comment]
Here's a couple of ideas on how we could "highlight" certain walkthroughs:

## Star adornment
We could add a little blue star in the corner (kind of like a bookmark) that would draw the user's attention. We can also show the description to show more context.
![image](https://user-images.githubusercontent.com/35271042/118171626-482eb980-b3e0-11eb-9dc2-09cf5a40a966.png)

## Label
Similarly we could add a "featured" label + show the description.
![image](https://user-images.githubusercontent.com/35271042/118171766-714f4a00-b3e0-11eb-8b5d-7ce1fdaa912d.png)

**Comment by misolori** [comment]
> what are your thoughts on the wireframes I posted, highlighting the first 3 walkthroughs in the top row? The mockups could also consider that all new users will have a (mostly) empty Recents list, so have more space available.

The biggest concern I have is how busy this will start to look. It already is starting to feel a bit busy as extensions are adding items there and making some items larger won't improve that. While it can make Day 0 feel better, i worry about the implications for Day 30 when you have recents + more extensions installed.

**Comment by digitarald** [comment]
Makes sense. Keeping that page simple and welcoming is key. The _star adornment_ is a good middle-ground:

1. Highlight first built-in walkthrough for new users until it is at least 50% done. The idea of half-completed is to prevent that users churn users against a 100% goal but also giving enough to do.
2. Highlight the last installed extension walkthrough that matches the current context.

@JacksonKearl is the latter feasible with the `when` clauses added to extensions? Right now it only supports the auto-open heuristic for the walkthroughs, right?

**Comment by meganrogge** [comment]
@digitarald @misolori can you move this to the June milestone?

**Comment by JacksonKearl** [comment]
Maybe we could show the star next to "new" contributions? i.e. record the first time we saw it, and if that's less than say a week ago show the star? This could apply to both built in and contributed walkthroughs.

**Comment by alefragnani** [comment]
I also prefer the _star adornment_, not only because it is less distracting, but also because it shares the same approach from `Extensions Viewlet`.
